kani-compiler/src/codegen_cprover_gotoc/context/goto_ctx.rs

```diff
@@ -153,7 +159,8 @@ impl<'tcx> GotocCtx<'tcx> {
         loc: Location,
     ) -> (Expr, Stmt) {
         let c = self.current_fn_mut().get_and_incr_counter();
-        let var = self.gen_stack_variable(c, &self.current_fn().name(), "temp", t, loc).to_expr();
+        let var =
+            self.gen_stack_variable(c, &self.current_fn().name(), "temp", t, loc, false).to_expr();
```

Comment:
Oh, a proliferation of boolean arguments isn't good either. Maybe ignore my comment here, or consider a "gen_hidden_local_variable" function or something like that.

Reply:
Sorry, I thought you were talking about the explicit `Symbol::variable` call in function.rs.

This does name variables "temp" instead of "var". I would rather not change that in this PR if that's OK for you.